projects/plugins/jetpack/class.jetpack.php

```diff
@@ -13,6 +13,7 @@
 use Automattic\Jetpack\Connection\Webhooks as Connection_Webhooks;
```

Comment:
Sync Package needs to be updated to call IDC version of sync_idc_optin and get_sync_error_idc_optin